asciidoc/volume1/conformance-statements/tf1-ch-c-conformance-statement-ieee-11073-10207-2017.adoc

```diff
@@ -6,7 +6,7 @@
 [%autowidth]
 [cols="1"]
 |===
-| *SDPi Supplement Version Note*: This section is provided as a placeholder for the <<acronym_ics>> table specification that will be added in the subsequent SDPi 1.2 supplement version.
+| *SDPi Supplement Version Note*: This section is provided as a placeholder for the <<acronym_ics>> table specification that will be added in the subsequent SDPi 1.3 supplement version.
```

Comment:
This should probably be saying "in the subsequent SDPi 1.4 supplement version". However for better maintainability I propose "in a future SDPi supplement version".